KatelynKim/iced-happy-latte.github.io
# Created a mood form with dynamically changing themes

### Summary
Created a mood form with dynamically changing themes. 
Depencides added:
  ```
 "@emotion/react": "^11.10.4",
  "@emotion/styled": "^11.10.4",
  "@mui/icons-material": "^5.10.9",
  "@mui/lab": "^5.0.0-alpha.104",
  "@mui/material": "^5.10.10",
```

### Steps to test
Change the value of the `mainWeather` propr inside App.jsx and see the theme changes.
